app/scripts/modules/google/src/serverGroup/configure/wizard/advancedSettings/advancedSettingsSelector.directive.js

```diff
@@ -48,5 +48,12 @@ module.exports = angular
           this.command.onHostMaintenance = 'MIGRATE';
         }
       };
+
+      this.setEnableVtpm = () => {
+        if (!this.command.enableVtpm) {
+          // Integrity monitoring requires vTPM to be enabled.
+          this.command.enableIntegrityMonitoring = false;
```

Comment:
Perhaps we could also disable the integrity monitoring checkbox unless vTPM is enabled?

Reply:
Done. Thanks.